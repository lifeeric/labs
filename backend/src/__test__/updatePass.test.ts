import * as util from "util";
import axios from "axios";

const API = "http://localhost:8000/graphql";
describe("updatePassword.ts", () => {
  it("should update password", async () => {
    const query = `mutation {
        updatePassword(id: "5f449b73e2ccbc43aa520488",oldPassword: "alex", newPassword: "alex", newPasswordAgain: "alex" ){
          error
          message
        }
      }`;

    const response = await axios.post(
      API,
      {
        query,
      },
      {
        headers: {
          Authorization:
            "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZjQ0OWI3M2UyY2NiYzQzYWE1MjA0ODgiLCJlbWFpbCI6ImFsZXhAdGVzdC5jb20iLCJpYXQiOjE1OTg1MjE0NDcsImV4cCI6MTYwMTExMzQ0N30.XixEJ5GnAvMZw784gWg1FThe3dLDVIGXNh-B8T-2-24",
        },
      }
    );

    expect(response.data).toMatchSnapshot();
  });
});
