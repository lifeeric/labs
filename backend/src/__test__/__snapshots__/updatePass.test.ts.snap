// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`updatePassword.ts should update password 1`] = `
Object {
  "data": Object {
    "updatePassword": Object {
      "error": true,
      "message": "Password successfuly changed!",
    },
  },
}
`;
